NekoGirlStateMachine.js

```
@@ -1,80 +1,83 @@
-const EventEmitter = require('events');
+const EventEmitter = require("events");
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)